README.md

```diff
@@ -8,6 +8,10 @@ Generates a Typescript type definition file for a C# POCO class.  Takes in a str
 - Gulp: https://github.com/ffMathy/gulp-typescript-cs-poco
 - Grunt: https://github.com/ffMathy/grunt-typescript-cs-poco
 
+## Installation with npm
+
+`npm install --save typescript-cs-poco`
```

Comment:
You should maybe consider `--save-dev` instead here? I assume this will always be used as a development tool.